NorthstarDLL/core/vanilla.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+/// Determines if we are in vanilla-compatibility mode.
+/// In this mode we shouldn't auth with Atlas, which prevents users from joining a
+/// non-trusted server. This means that we can unrestrict client/server commands
+/// as well as various other small changes for compatibility
+class VanillaCompatibility
```

Comment:
game dev try to not make everything a class challenge

Reply:
I did this for two reasons:
1. Everything else does it
2. I want it only to be set once